src/Controls/samples/Controls.Sample.UITests/Issues/Issue17022.xaml.cs

```diff
@@ -0,0 +1,46 @@
+﻿using Microsoft.Maui.Controls;
+using Microsoft.Maui.Controls.Xaml;
+using Microsoft.Maui.Graphics;
+using Microsoft.Maui.Controls.PlatformConfiguration;
+using Microsoft.Maui.Controls.PlatformConfiguration.iOSSpecific;
+
+namespace Maui.Controls.Sample.Issues;
+
+[XamlCompilation(XamlCompilationOptions.Compile)]
+[Issue(IssueTracker.Github, 17022, "UINavigationBar is Translucent", PlatformAffected.iOS)]
+public partial class Issue17022 : ContentPage
+{
+	Color _initialBarBackgroundColor;
+	bool _initialTranslucent;
+
+	public Issue17022()
+	{
+		InitializeComponent();
+	}
+
+	protected override void OnAppearing()
```

Comment:
For this test can we push a test page modally? 

So push a NavigationPage modally and do the validation against that? This way we aren't modifying the test runner chrome?